memory/src/main/java/org/opensearch/ml/memory/index/InteractionsIndex.java

```diff
@@ -71,28 +73,23 @@ public class InteractionsIndex {
     private Client client;
     private ClusterService clusterService;
     private ConversationMetaIndex conversationMetaIndex;
+    private final String indexName = ConversationalIndexConstants.INTERACTIONS_INDEX_NAME;
```

Comment:
Why use another variable to wrap a static constant `ConversationalIndexConstants.INTERACTIONS_INDEX_NAME` ?

Reply:
Removed. This is a leftover from the old version in the feature branch. I see it's updated in the main already.